index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
! DOCTYPE html  what work of it

Reply:
It tells the browser that this is an html document.